pkgs/servers/misc/navidrome/default.nix

```diff
@@ -52,7 +58,9 @@ buildGoModule {
       --prefix PATH : ${lib.makeBinPath [ ffmpeg ]}
   '';
 
-  doCheck = false;
+  preCheck = ''
+    export GOFLAGS=''${GOFLAGS//-trimpath/}
```

Comment:
Can you explain what this line fixes?

Reply:
When using `-trimpath`, the unit tests can't find any of their needed files, so they fail.

Reply:
We use trimpath to remove references to the go compiler in the resulting build: 

https://github.com/NixOS/nixpkgs/blob/2f0c3be57c348f4cfd8820f2d189e29a685d9c41/pkgs/development/go-modules/generic/default.nix#L145

Reply:
But for tests, there should be no harm in having those references.

Reply:
cc @zowoq

Reply:
https://github.com/NixOS/nixpkgs/pull/182665

Reply:
> But for tests, there should be no harm in having those references.

Yes, should be fine for tests as we also have `disallowedReferences` for `go`.